.github/workflows/spread-large.yaml

```diff
@@ -47,4 +42,4 @@ jobs:
           name: snap
       - name: Spread k8s operator
         run: |
-          spread google:ubuntu-22.04-64:tests/spread/charms/${{ matrix.charm }}
+          spread google tests/spread/charms/*
```

Comment:
`google tests` -> `google:tests` ?

Reply:
Surprisingly not (I tried that first) :-)

This creates an `AND` on the filters, filtering for the google provider and that directory.

Since I still have it in my terminal history:

```
lengau@odyssey:~/Work/charmcraft$ spread -list google:tests/spread/charms/*
error: nothing matches provider filter
lengau@odyssey:~/Work/charmcraft$ spread -list google::tests/spread/charms/*
error: invalid filter string: "::"
lengau@odyssey:~/Work/charmcraft$ spread -list google:*:tests/spread/charms/*
error: invalid filter string: "google:*:tests/spread/charms/*"
lengau@odyssey:~/Work/charmcraft$ spread -list google tests/spread/charms/*
google:ubuntu-18.04-64:tests/spread/commands/version
google:ubuntu-18.04-64:tests/spread/smoketests/basic-centos-7
google:ubuntu-20.04-64:tests/spread/commands/version
google:ubuntu-20.04-64:tests/spread/smoketests/basic
google:ubuntu-20.04-64:tests/spread/smoketests/destructive-mode
google:ubuntu-20.04-64:tests/spread/smoketests/different-dependencies
google:ubuntu-20.04-64:tests/spread/smoketests/different-dir
google:ubuntu-20.04-64:tests/spread/smoketests/full-lifecycle
google:ubuntu-20.04-64:tests/spread/smoketests/reactive
google:ubuntu-22.04-64:tests/spread/commands/version
google:ubuntu-22.04-64:tests/spread/hooks/configure
google:ubuntu-22.04-64:tests/spread/hooks/remove
google:ubuntu-22.04-64:tests/spread/smoketests/basic
google:ubuntu-22.04-64:tests/spread/smoketests/destructive-mode
google:ubuntu-22.04-64:tests/spread/smoketests/different-dependencies
google:ubuntu-22.04-64:tests/spread/smoketests/different-dir
google:ubuntu-22.04-64:tests/spread/smoketests/full-lifecycle
google:ubuntu-22.04-64:tests/spread/smoketests/reactive
```

Reply:
Ahhh yes you're right. Moving this back to draft to fix